lib/Mojolicious/Guides/Cookbook.pod

```diff
@@ -342,6 +342,51 @@ web features, such as WebSockets, are not available.
 
   ScriptAlias / /home/sri/my_app/script/my_app/
 
+=head2 Envoy Proxy
```

Comment:
Why does the header need a "Proxy" qualifier? Are there different ways to deploy Mojolicious apps with Envoy?

Reply:
Looking at the [README](https://github.com/envoyproxy/envoy) it's just referred to as Envoy and not Envoy Proxy. I don't think the "Proxy" is actually part of the name.

Reply:
Doesn't matter now, its already dropped :muscle: